core/store/orm/orm.go

```diff
@@ -217,6 +217,13 @@ func (orm *ORM) FindBridge(name models.TaskType) (models.BridgeType, error) {
 	return bt, orm.db.First(&bt, "name = ?", name.String()).Error
 }
 
+// FindBridgesByNames finds multiple bridges by their names.
+func (orm *ORM) FindBridgesByNames(names []string) ([]models.BridgeType, error) {
```

Comment:
What happens if a name is included that doesn't actually exist? Will this error? Should check the length of `bt` is the same as `names`?

Reply:
In `validators.go` (the only place this function is used) I do check that len(names) == len(bridges). Wasn't sure if where would make the most sense, but happy to move that check to the orm if you think it's better here (I kind of agree with that idea - let's throw the error as early as possible).

Reply:
How come you opted for an array argument instead of a single one?  It would give more control to the caller about any issue about a single bridge.

Reply:
not sure what you mean here @felder-cl. Agreed though that it would be nice to have errors about specifically *which* bridges we were unable to find.

Reply:
If there are duplicates, like `['bnc', 'bnc']`, will it return multiple? I think we may want to support that, but could split it out into another story.

Reply:
ahh yeah good point - right now it should only return one instance because the query would be some thing like...
```sql
WHERE name IN (bnc, bnc)
```
which would still only yield a single matching row. I think steve's idea of requiring the length be the same here would make this function more robust and address that issue as well.

Reply:
I'm not sure we need multiple of the same feed, but it seems like an easy way to give weight to feeds, so it would be nice to have. Was not in the original requirements, we could break it out into another story if it's too much scope creep here.

Reply:
What about validating that the feeds are unique to simplify our assumptions, then following up with a story which we can use as a focal point for discussion?

Reply:
Good idea. Limits scope and cuts edge cases.